library-model/test-library-model-generator/src/main/java/com/uber/nullaway/libmodel/AnnotationExample.java

```diff
@@ -39,4 +39,12 @@ public String returnNull() {
       return null;
     }
   }
+
+  /** In the library model we add a {@code @Nullable} upper bound for T */
+  public static class UpperBoundExample<T> {
+
+    public T getNull() {
+      return null;
+    }
+  }
```

Comment:
I'm not suggesting to change the return type to `@Nullable T`.  I'm suggesting to have a field of type `T` and to return the value of that field, rather than just returning `null`.  Then the return type `T` would be appropriate.

Reply:
Oh that makes sense, thanks!